.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "Auto-GPT"]
+	path = agent/Auto-GPT
+	url = https://github.com/Significant-Gravitas/Auto-GPT.git
+	branch = benchmark-integration
```

Comment:
temporary branch, soon it will be master. I need to merge this benchmark-integration branch into the main repo